src/Components/programs/Program.js

```diff
@@ -1,41 +1,39 @@
 import React from 'react';
-import { StyleSheet, TouchableOpacity, View, Text, Image } from 'react-native';
+import { StyleSheet, View, Text, ImageBackground } from 'react-native';
 import { connect } from 'react-redux';
+import { LinearGradient } from 'expo-linear-gradient';
 
 import theme from '../../styles/theme.style';
-import ListItem from '../shared/ListItem';
-import arrowDark from '../../../assets/arrow_dark.png';
 
 export const Program = props => {
-  const { title, trainings } = props.program;
+  const { title, trainings, image } = props.program;
 
   const completeTrainingsCount = props.completeTrainings
     .filter(({ training, program }) => program.id === props.program.id)
     .map(({ training, program }) => training).length;
 
-  const onPress = () => {
-    props.navigation.navigate('ProgramPage', { program: props.program });
-  };
-
   const width = `${(completeTrainingsCount * 100) / trainings.length}%`;
   return (
-    <ListItem>
-      <TouchableOpacity style={styles.program} onPress={onPress}>
-        <View style={styles.presentationContainer}>
+    <View style={styles.program}>
+      <ImageBackground source={{ uri: image }} style={styles.image} imageStyle={styles.imageOpacity}>
```

Comment:
I'm not a big fan of the background image on the programs. It feels overloaded, the screen is full of information

I would prefer to keep the minimalist UI we have right now (which I admit is not perfect)

Reply:
I am ok with your choice. I will remove the imagebackground. I still have an issue with the flatlist inside the scrollview. I will be looking for a solution today